src/renderer/components/NodeDocumentation/NodeExample.tsx

```diff
@@ -150,6 +151,29 @@ export const NodeExample = memo(({ accentColor, selectedSchema }: NodeExamplePro
         };
     }, [selectedSchema]);
 
+    const nodeState: NodeState = {
+        id: nodeId,
+        schemaId: selectedSchema.schemaId,
+        schema: selectedSchema,
+        inputData,
+        setInputValue,
+        inputHeight,
+        nodeWidth,
+        outputHeight,
+        setOutputHeight: setSingleOutputHeight,
+        setWidth,
+        setInputHeight: setSingleInputHeight,
+        isLocked: false,
+        connectedInputs: EMPTY_SET,
+        connectedOutputs: EMPTY_SET,
+        iteratedInputs,
+        iteratedOutputs,
+        type: typeInfo,
+        testCondition: testForInputConditionTypeInfo(inputData, selectedSchema, typeInfo),
+    };
+
+    const [isCollapsed, setIsCollapsed] = useState(false);
```

Comment:
What's the point of being able to minimize nodes in node documentation? I think example nodes shouldn't have the collapse toggle.

Reply:
I thought we wanted the nodes in the docs to behave just like they do in the editor for demonstration purposes?

Reply:
For inputs and outputs, yes. But I don't see any reason why nodes should be collapsible in our docs.

Reply:
Ok